src/api/common/HttpClient.ts

```diff
@@ -70,7 +71,25 @@ export default class HttpClient {
 
     return response;
   }
+
+  async getPingResponse(
+    relayUrl: string,
+    verifier?: string
+): Promise<PingResponse> {
+    const verifierSuffix = verifier == null ? '' : '?verifier=' + verifier;
+    const pingResponse = await this._httpWrapper.sendPromise<PingResponse>(
+        relayUrl + '/getaddr' + verifierSuffix
+    );
+    if (pingResponse == null) {
+        throw new Error('Relay responded without a body');
+    }
+    log.info(`pingResponse: ${JSON.stringify(pingResponse)}`);
+
+    return pingResponse;
+  }
```

Comment:
this is already implemented in [getChainInfo](https://github.com/rsksmart/rif-relay-client/pull/49/files#diff-17fc46d2093dfe0332d5af3ff266138a5ea4f1bb0f2cb7afa316fb2a122fb1fdL30-L39) isn't it?

Reply:
Yes, you're right. Done.